stylesheet.css

```diff
@@ -0,0 +1,69 @@
+body {
+    background-color: #f1f1f1;
+    font-family: calibri;
+    color:#5f5f5f;
+}
+
+header {
+  background: #5f5f5f;
+    padding: 1em;
+}
+
+header a {
+    color: white;
+    margin: 2em;
+    text-decoration:none;
+    
+}
```

Comment:
the same with separation lines between the blocks. I prefer to have one empty line between each css rule